tests/test_HNL.py

```diff
@@ -1,12 +1,13 @@
 import os
 
-import pytest
-
 import numpy as np
 from numpy.testing import assert_array_almost_equal, assert_array_equal
 
+import pytest
```

Comment:
A small NIT: I actually have a problem with the order of the imports, but to stick with convention now might cause a lot of work.

> Imports should be grouped in the following order:
>
>    standard library imports
>    related third party imports
>    local application/library specific imports

From [PEP8 on Imports](https://www.python.org/dev/peps/pep-0008/#imports).

So this would be
```python
import os

import numpy as np
import pytest
from numpy.testing import assert_array_almost_equal, assert_array_equal

import utide
from utide.utilities import Bunch, convert_unicode_arrays, loadbunch
```

Like I said above, doing this everywhere would be a bit of work. I may do this later at some point

Reply:
I like `flake8-import-order` for that but I don't think it should be a priority here. It is a minor styling issue.